content/influxdb/cloud-dedicated/get-started/query.md

```diff
@@ -364,24 +380,21 @@ _If your project's virtual environment is already running, skip to step 3._
   2.  In your Python code, import `certifi` and call the `certifi.where()` method to retrieve the certificate path.
   3.  When instantiating the client, pass the `flight_client_options.tls_root_certs=<ROOT_CERT_PATH>` option with the certificate path--for example:
 
-      ```py
+      ```python
       from influxdb_client_3 import InfluxDBClient3, flight_client_options
       import os
       import certifi
 
-      TOKEN = os.getenv('INFLUX_TOKEN')
-
       fh = open(certifi.where(), "r")
       cert = fh.read()
       fh.close()
 
       client = InfluxDBClient3(
-          host="{{< influxdb/host >}}",
-          token=TOKEN,
-          database="get-started",
+          host=f"{{< influxdb/host >}}",
+          token=f"DATABASE_TOKEN",
```

Comment:
Where these are being replace with "static" values, do these need to be f-strings? They shouldn't need any interpolation unless they're assigning these to a local or environment variable.